cours/containers/kubernetes/k8s-architecture.fr.md

```diff
@@ -0,0 +1,132 @@
+# Kubernetes : Architecture {-}
```

Comment:
why {-} here?

Reply:
i wanted to simulate a behavior. forgot to remove it. thanks.